src/view/Champion/ChampionPortrait.jsx

```diff
@@ -63,11 +63,9 @@ const ChampionPortrait = {
             }
             if(showPi && (pi || champion.pi)) {
                 title.push(
-                    <div
-                        class={ classNames('title-field', 'title-field-pi', {
-                            'title-field-pi-custom': pi && pi > 0,
-                        }) }
-                    >{ lang.number(pi || champion.pi * scalePi) }</div>
+                    <div class={ classNames('title-field', 'title-field-pi', {
+                        'title-field-pi-custom': pi && pi > 0 }, 'title-field-sig') }
+                    >{ lang.number(pi || champion.pi * scalePi) + lang.string(` sig: ${awakened}`)}</div>
```

Comment:
If we want to show sig, instead do it like:

```js
{ `${lang.number(pi || champion.pi * scalePi)} ${lang.string('portrait-signature')}: ${awakened}` }
```

and then just set `portrait-signature` to "sig"